soundbay/data.py

```diff
@@ -151,7 +151,9 @@ def _set_augmentations(self, augmentations_dict, augmentations_p):
             augmentations_list = [instantiate(args) for args in augmentations_dict.values()]
         else:
             augmentations_list = []
-        augmenter = ChainedAugmentations(augmentations_list, augmentations_p) if self.mode == 'train' else torch.nn.Identity()
+        augmenter = lambda x: torch.tensor(
+            Compose(augmentations_list, p=augmentations_p, shuffle=True)(x, self.sample_rate)
```

Comment:
Nice, does the `shuffle` feature here should be something we should attend compared to previous version?

Reply:
I think there is no harm in adding this, but we don't have to. Can start without it